PicoStreamingAssistantFTUDP/Pico4SAFTExtTrackingModule.cs

```diff
@@ -25,23 +25,37 @@ public sealed class Pico4SAFTExtTrackingModule : ExtTrackingModule, IDisposable
 
     public override (bool SupportsEye, bool SupportsExpression) Supported { get; } = (true, true);
 
-    private bool StreamerValidity()
+    private bool StreamerValidity(bool isInitialize)
     {
-        this.connector = ConnectorFactory.build(Logger, new ProcessRunningProgramChecker(), new ConfigChecker(Logger));
-        if (this.connector == null)
+        if (isInitialize)
         {
-            Logger.LogError("\"Streaming Assistant\", \"Streaming Assistant\" or \"PICO Connect\" process was not found. Please run the Streaming Assistant or PICO Connect before VRCFaceTracking.");
-            return false;
-        }
+            this.connector = ConnectorFactory.build(Logger, new ProcessRunningProgramChecker(), new ConfigChecker(Logger));
+            if (this.connector == null)
+            {
+                Logger.LogError("\"Streaming Assistant\", \"Streaming Assistant\" or \"PICO Connect\" process was not found. Please run the Streaming Assistant or PICO Connect before VRCFaceTracking.");
+                return false;
+            }
 
-        Logger.LogInformation("Using {}.", this.connector.GetProcessName());
-        return true;
+            Logger.LogInformation("Using {}.", this.connector.GetProcessName());
+            return true;
+        }
+        else
+        {
+            var programChecker = new ProcessRunningProgramChecker();
+            bool using_sa = programChecker.Check(PicoPrograms.StreamingAssistant);
```

Comment:
This basically is code duplication of ConnectorFactory, and I'd rather not do that. Also, at first sight I don't see a difference between both branches... I'll have to check exactly why this solves the issue